windows_containers/understanding-windows-container-workloads.adoc

```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Windows Container Support for Red Hat OpenShift provides built-in support for running Microsoft Windows Server containers on {product-title}. For those that administer heterogeneous environments with a mix of Linux and Windows workloads, {product-title} allows you to deploy Windows workloads running on Windows Server containers while also providing traditional Linux workloads hosted on {op-system-first} or {op-system-base-full}.
+Red Hat OpenShift support for Windows Containers provides built-in support for running Microsoft Windows Server containers on {product-title}. For those that administer heterogeneous environments with a mix of Linux and Windows workloads, {product-title} allows you to deploy Windows workloads running on Windows Server containers while also providing traditional Linux workloads hosted on {op-system-first} or {op-system-base-full}.
```

Comment:
You have the attributes that define the name, shouldn't all of these hardcoded instances be updated to use the attributes?

Reply:
@bergerhoffer I added a new attribute in place of the hard-coded name. Can you PTAL?